imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -22,14 +20,21 @@ class CaseExplorer extends Component {
       caseId: '',
       expandedUnits: [],
       unitsDict: {},
-      showOpen: true
+      showOpen: true,
+      assignedToMe: null
     }
   }
 
   handleStatusClicked (value) {
     this.setState({ showOpen: value })
   }
 
+  handleAssignedClicked () {
+    const currentUser = this.props.currentUser.bugzillaCreds.login
+    const change = this.state.assignedToMe ? null : currentUser
+    this.setState({ assignedToMe: change })
```

Comment:
It will simpler and easier to understand if assignedToMe is just a boolean rather than the user object... It is better to for the state and mutable properties in general to explicitly describe what they stand for, so since `assignedToMe` is designed to be either on or off, you can see how it just makes much more sense if it's a boolean